js_modules/dagster-ui/packages/ui-core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -67,14 +67,17 @@ export const SidebarAssetInfo = ({graphNode}: {graphNode: GraphNode}) => {
   const {lastMaterialization} = liveData || {};
   const asset = data?.assetNodeOrError.__typename === 'AssetNode' ? data.assetNodeOrError : null;
 
-  const recentEvents = useRecentAssetEvents(
-    asset?.assetKey,
-    1,
-    MaterializationHistoryEventTypeSelector.MATERIALIZATION,
+  const recentEvents = useRecentAssetEvents(asset?.assetKey, 1, [
+    AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+    AssetEventHistoryEventTypeSelector.OBSERVATION,
+  ]);
+  const materializations = recentEvents.events.filter(
+    (event) => event.__typename === 'MaterializationEvent',
   );
-  const latestMaterializationEvent = recentEvents.materializations
-    ? recentEvents.materializations[recentEvents.materializations.length - 1]
-    : undefined;
+
+  const latestMaterializationEvent = materializations
+    ? materializations[materializations.length - 1]
+    : undefined; // TODO shouldn't this be the first one in the list, not last? I thought the list was descending timestamp (newer first)
```

Comment:
It's a bit odd but in JS empty arrays are truthy (unlike python), so this should always use the first path through this ternary.

I think that's ok though!